mirdata/datasets/maestro.py

```diff
@@ -147,21 +147,36 @@ def __init__(self, track_id, data_home):
 
     @utils.cached_property
     def midi(self):
-        """output type: description of output"""
+        """Description of output.
+
+        Retuns:
+            (Obj): midi"""
         return load_midi(self.midi_path)
 
     @utils.cached_property
     def notes(self):
-        """NoteData: annotated piano notes"""
+        """Annotated piano notes
+
+        Returns:
+            (NoteData): notes
+        """
         return load_notes(self.midi_path, self.midi)
 
     @property
     def audio(self):
-        """(np.ndarray, float): track's audio signal, sample rate"""
+        """Audio ground truth
+
+        Returns:
+            (np.ndarray, float): audio signal, sample rate
```

Comment:
For all the audio properties, I think it will be more clear to document it like:

```        
Returns:
    (np.ndarray): audio signal
    (float): sample rate
```

Reply:
this error is in the load_audio methos too... I also change it....

Reply:
done